Fabric.Authorization.AccessControl/src/app/services/global/services.service.ts

```diff
@@ -49,35 +48,12 @@ export class ServicesService {
     constructor(private http: HttpClient, private configService: ConfigService) { }
 
     public initialize() {
-        // prevent discovery from being called twice with oauth disabled
-        this.isOAuthAuthenticationEnabled.toPromise().then(
-            isEnabled => {
-                if (isEnabled) {
-                    this.buildServiceMaps().toPromise();
-                }
-            }
-        );
-    }
-
-    public getIdentityAndAccessControlUrl(): Observable<UrlResponse> {
         return this.isOAuthAuthenticationEnabled.pipe(
             switchMap(isEnabled => {
-              this.services.find(s => s.name === 'DiscoveryService').requireAuthToken = isEnabled;
-              if (isEnabled) {
-                // if OAuth, we get identity and access control up front
-                return forkJoin(this.identityServiceEndpoint, this.accessControlEndpoint);
-              } else {
-                // If Windows Auth, we have discovery up front and have to build
-                // out the service urls from discovery
-                return this.buildServiceMaps().toPromise().then(identityUrl => {
```

Comment:
Dont forget about this scenario, if windows auth is enabled, then we have to pull the urls from discovery service